### ISSUE TITLE
Some questions about the codes

### ISSUE THREAD

**Issue by HongduanTian** [dragen1860/MAML-Pytorch#43]
1. About validation accuracy of MiniImageNet in MAML original code, the evaluation is performed on the validation set instead of the test set, but in this code, evaluation is performed on the test set.

2. There is no test phase in this code.

3. For the optimization, I think the auto-grad seems right. During the inner loop, it is a first-order optimization, and after evaluating on query data, the "net.parameters" is updated with Adam optimizer, which is the same as that in MAML code.
